GrandCentralBoard/Extensions/Alamofire+NetworkRequestManager.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Alamofire+NetworkRequestManager.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 07.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Alamofire
+
+extension Manager : NetworkRequestManager {
+
+    func requestJSON(method: Method, url: NSURL, parameters: [String : AnyObject]?, headers: [String : String]?, completion: (ResultType<AnyObject, NSError>.result) -> Void) {
+        let method = Alamofire.Method(rawValue: method.rawValue)!
```

Comment:
I'd suggest adding extension to `Alamofire.Method`
```swift
extension Alamofire.Method {
    init(fromMethod method: Method) {
        switch method {
        case .CONNECT: self = .CONNECT
        case .DELETE: self = .DELETE
        case .GET: self = .GET
        case .HEAD: self = .HEAD
        case .OPTIONS: self = .OPTIONS
        case .PATCH: self = .PATCH
        case .POST: self = .POST
        case .PUT: self = .PUT
        case .TRACE: self = .TRACE
        }
    }
}
```
Then we can use `let method = Alamofire.Method(fromMethod: method)`

We avoid force unwrapping, and fon't have to worry about either of enums adding/removing `case`s.